levels/menu/leveldata.c

```diff
@@ -1520,167 +1327,476 @@ ALIGNED8 static const Texture texture_menu_font_char_period[] = {
+    {0x30DB, 9, 0, texture_menu_font_char_jp_katakana_ho}, // ホ
+    {0x30DC, 9, TEXT_DIACRITIC_DAKUTEN, texture_menu_font_char_jp_katakana_ho}, // ボ
+    {0x30DD, 9, TEXT_DIACRITIC_HANDAKUTEN, texture_menu_font_char_jp_katakana_ho}, // ポ
+    {0x30DE, 9, 0, texture_menu_font_char_jp_katakana_ma}, // マ
+    {0x30DF, 9, 0, texture_menu_font_char_jp_katakana_mi}, // ミ
+    {0x30E0, 9, 0, texture_menu_font_char_jp_katakana_mu}, // ム
+    {0x30E1, 9, 0, texture_menu_font_char_jp_katakana_me}, // メ
+    {0x30E2, 9, 0, texture_menu_font_char_jp_katakana_mo}, // モ
+    {0x30E3, 9, 0, texture_menu_font_char_jp_katakana_small_ya}, // ャ
+    {0x30E4, 9, 0, texture_menu_font_char_jp_katakana_ya}, // ヤ
+    {0x30E5, 9, 0, texture_menu_font_char_jp_katakana_small_yu}, // ュ
+    {0x30E6, 9, 0, texture_menu_font_char_jp_katakana_yu}, // ユ
+    {0x30E7, 9, 0, texture_menu_font_char_jp_katakana_small_yo}, // ョ
+    {0x30E8, 9, 0, texture_menu_font_char_jp_katakana_yo}, // ヨ
+    {0x30E9, 9, 0, texture_menu_font_char_jp_katakana_ra}, // ラ
+    {0x30EA, 9, 0, texture_menu_font_char_jp_katakana_ri}, // リ
+    {0x30EB, 9, 0, texture_menu_font_char_jp_katakana_ru}, // ル
+    {0x30EC, 9, 0, texture_menu_font_char_jp_katakana_re}, // レ
+    {0x30ED, 9, 0, texture_menu_font_char_jp_katakana_ro}, // ロ
+    {0x30EF, 9, 0, texture_menu_font_char_jp_katakana_wa}, // ワ
+    {0x30F2, 9, 0, texture_menu_font_char_jp_katakana_wo}, // ヲ
+    {0x30F3, 9, 0, texture_menu_font_char_jp_katakana_n}, // ン
+    {0x30FC, 9, 0, texture_menu_font_char_jp_long_vowel}, // ー
+
+    {0xFF01, 9, 0, texture_menu_font_char_jp_exclamation}, // ！
+    {0xFF10, 9, 0, texture_menu_font_char_jp_0}, // ０
+    {0xFF11, 9, 0, texture_menu_font_char_jp_1}, // １
+    {0xFF12, 9, 0, texture_menu_font_char_jp_2}, // ２
+    {0xFF13, 9, 0, texture_menu_font_char_jp_3}, // ３
+    {0xFF14, 9, 0, texture_menu_font_char_jp_4}, // ４
+    {0xFF15, 9, 0, texture_menu_font_char_jp_5}, // ５
+    {0xFF16, 9, 0, texture_menu_font_char_jp_6}, // ６
+    {0xFF17, 9, 0, texture_menu_font_char_jp_7}, // ７
+    {0xFF18, 9, 0, texture_menu_font_char_jp_8}, // ８
+    {0xFF19, 9, 0, texture_menu_font_char_jp_9}, // ９
+    {0xFF21, 9, 0, texture_menu_font_char_jp_A}, // Ａ
+    {0xFF22, 9, 0, texture_menu_font_char_jp_B}, // Ｂ
+    {0xFF23, 9, 0, texture_menu_font_char_jp_C}, // Ｃ
+    {0xFF24, 9, 0, texture_menu_font_char_jp_D}, // Ｄ
 #endif
 };
 
+const struct Utf8CharLUTEntry menu_font_utf8_4byte_lut[] = {
+
+};
+
+const struct Utf8CharLUTEntry menu_font_utf8_missing_char = {0, 8, 0, texture_menu_font_missing_character};
+
+const struct Utf8LUT menu_font_utf8_lut = {
+    menu_font_utf8_2byte_lut,
+    menu_font_utf8_3byte_lut,
+    menu_font_utf8_4byte_lut,
+    ARRAY_COUNT(menu_font_utf8_2byte_lut),
+    ARRAY_COUNT(menu_font_utf8_3byte_lut),
+    ARRAY_COUNT(menu_font_utf8_4byte_lut),
+    &menu_font_utf8_missing_char,
+};
+
+// Menu small font print table
```

Comment:
whats all this commented out junk for? can it be deleted? or at least a comment explaining why its here

Reply:
not gonna lie i totally just forgot to remove it